o2a/templates/prepare.tpl

```diff
@@ -12,10 +12,24 @@
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
- #}
+#}
 
 {{ task_id | to_var }} = bash_operator.BashOperator(
-    task_id={{ task_id | tojson }},
-    trigger_rule={{ trigger_rule | tojson }},
-    bash_command={{ prepare_command | tojson }},
+    task_id={{ task_id | to_python }},
+    trigger_rule={{ trigger_rule | to_python }},
+{#
+ In BashOperator in Composer we can't read from $DAGS_FOLDER (~/dags) - permission denied.
+ However we can read from ~/data -> /home/airflow/gcs/data.
+ The easiest way to access it is using the $DAGS_FOLDER env variable.
+#}
+    bash_command="$DAGS_FOLDER/../data/prepare.sh "
```

Comment:
I think this should be in the `prepare_command.tpl file to keep one style.

Reply:
Good idea. Split in 81758e1c3b33e048bc1b4b471ca1e52818b4d451